### PR TITLE
Console: Fixed issues involving Console History not being capped at specified limit, regarding Issue #10598

### LINKED ISSUE #10598: Console History is not trimmed to maximum
### Bug Description:

The history of commands typed in the debug console will not be capped at the intended limit (50), but grows forever).

### Steps to Reproduce:

1. To test, ideally first lower the limit from 50 to something lower, e.g. 3 in: https://github.com/eclipse-theia/theia/blob/8083f76b4f4969bad12549ebba0a2c1df9591552/packages/console/src/browser/console-history.ts#L22
2. Open Theia and the debug console.
3. Execute more commands then the limit (e.g. 4)
4. Press arrow up and see that more commands than then limit are stored in the history

### Additional Information

Looks like the history array is sliced correctly in the trim function, but the return value is ignored. The return value should be assigned to the original array, see https://github.com/eclipse-theia/theia/blob/8083f76b4f4969bad12549ebba0a2c1df9591552/packages/console/src/browser/console-history.ts#L42

### PR DESCRIPTION
**Pull request by ShehanAT**
#### What it does
The history of commands typed in the Debug Console in the Debug view(access in the left sidebar of the browser version) was not adhering to the limit specified in the `limit` variable in the file: `theia/packages/console/src/browser/console-history.ts`. 

The main issue was that the newly sliced array(`this.values.slice(index)`) was not being assigned to the original array. 

Therefore, I replaced that line with the following: `this.values = this.values.slice(index);` in order to resolve this issue.

#### How to test
1. Navigate to the file: `theia/packages/console/src/browser/console-history.ts`
2. Change the `limit` variable to a lower number such as `3` 
3. Build the browser version via `yarn build browser` then navigate to `/examples/browser` and run `yarn start`
4. Navigate to `http://localhost:3000` and open the `Debug` tab in the left sidebar
5. Enter many commands(at least over the number specified for the `limit` variable) in the debug console then press the up arrow to validate that the messages saved conforms to the number assigned to the `limit` variable(Ex: `limit = 3` should cap the maximum number of saved commands up to 3)

#### Review checklist

- [X] As an author, I have thoroughly tested my changes and carefully followed [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#requesting-a-review)

#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,10 @@
 
 - [Previous Changelogs](https://github.com/eclipse-theia/theia/tree/master/doc/changelogs/)
 
+
+## v.1.27.0
+- [console] fixed issue in Debug console where console history was not being trimmed in accordance with the maximum commands limit [#10598](https://github.com/eclipse-theia/theia/pull/10598)
```
Comment (vince-fugnitto) [inline CHANGELOG.md:9]: The next release would be [`1.26.0`](https://github.com/eclipse-theia/theia/releases) not `1.27.0` so we can move the item down to `1.26.0`.

File: examples/browser/webpack.config.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-/**
+AAAA/**
```
Comment (vince-fugnitto) [inline examples/browser/webpack.config.js:1]: Please revert.

**Review by vince-fugnitto — COMMENTED** [review]
Thank you for your contribution :+1: 

In order to accept your changes please be sure to sure to sign the [eclipse contributor agreement (eca)](https://github.com/eclipse-theia/theia/blob/master/CONTRIBUTING.md#eclipse-contributor-agreement) with the same email as your authorship.

File: CHANGELOG.md (hunk 1 above)
Comment (ShehanAT) [reply CHANGELOG.md]: Ok, this is updated in the next commits

File: examples/browser/webpack.config.js (hunk 2 above)
Comment (ShehanAT) [reply examples/browser/webpack.config.js]: Sorry about that, removed in the next commits now

**Comment by ShehanAT** [comment]
> Thank you for your contribution 👍
> 
> In order to accept your changes please be sure to sure to sign the [eclipse contributor agreement (eca)](https://github.com/eclipse-theia/theia/blob/master/CONTRIBUTING.md#eclipse-contributor-agreement) with the same email as your authorship.

I have signed to Eclipse Contributor Agreement now.

**Review by JonasHelming — APPROVED** [review]
Thank you for your contribution!
I will  @vince-fugnitto have a final look and then we can merge this one!

**Review by vince-fugnitto — APPROVED** [review]
Thank you for your first contribution @ShehanAT 👍

**Pull request merged**